### PR TITLE
Docs for authorization feature

### PR DESCRIPTION
**Pull request by israel-hdez**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:
-->
```release-note
NONE
```

### CODE REVIEW

File: docs/odh/authorino-setup-manual.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+```yaml
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+   name: kserve-predictor
+spec:
+   action: CUSTOM
+   provider:
+      name: opendatahub-auth-provider
```
Comment (Jooho) [inline docs/odh/authorino-setup-manual.md:103]: Where can I find this envoyfilter ?

Comment (israel-hdez) [reply docs/odh/authorino-setup-manual.md:103]: It is not an envoyfilter, it is a configuration in the ServiceMeshControlPlane. See: https://github.com/opendatahub-io/kserve/pull/253/files#diff-649512b8d7a6b58f996c1a23ee4e257cf224998c7ef287cc3b722b8f8c07ddcbR71

File: docs/odh/authorino-setup-manual.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+needs to be modified to configure Authorino as an authorization provider of the Mesh.
+
+Assuming you have your `ServiceMeshControlPlane` resource in the `istio-system` namespace,
+you named it `data-science-smcp`, and you followed the previous section without
+renaming any resource, the following is a patch that you can apply with
+`oc patch smcp --type merge -n istio-system`:
+
+```yaml
+spec:
+  techPreview:
```
Comment (Jooho) [inline docs/odh/authorino-setup-manual.md:68]: curious. Is it still techpreview?

Comment (israel-hdez) [reply docs/odh/authorino-setup-manual.md:68]: I think yes.

**Pull request merged**